### DISCUSSION TITLE
Is there a way to add a QR Code for AR button initializer?

### DISCUSSION THREAD

**Discussion by glitch-txs** [pmndrs/xr#178] [Q&A] [answered]
I'm working on a three.js website I'd like to make it AR only, is there a way that I can set a button for PC users so they click on it and a QR code shows up to open the AR in their phones?

**Comment by CodyJasonBennett** [comment, marked as answer]
There currently isn't marker integration in react-xr. I'd suggest https://github.com/AR-js-org/AR.js/ for the time being.
